django_tenants/middleware/main.py

```diff
@@ -54,7 +55,17 @@ def process_request(self, request):
     def no_tenant_found(self, request, hostname):
         """ What should happen if no tenant is found.
         This makes it easier if you want to override the default behavior """
-        if hasattr(settings, 'SHOW_PUBLIC_IF_NO_TENANT_FOUND') and settings.SHOW_PUBLIC_IF_NO_TENANT_FOUND:
+        if hasattr(settings, 'DEFAULT_NOT_FOUND_TENANT_VIEW'):
+            view_path = settings.DEFAULT_NOT_FOUND_TENANT_VIEW
+            view = import_string(view_path)
+            if hasattr(view, 'as_view'):
+                response = view.as_view()(request)
+            else:
+                response = view(request)
+            if hasattr(response, 'render'):
+                response.render()
```

Comment:
Question: if I want to display something from the DB what's the current schema context in which this custom view operates? 

Seems to be a bit undefined (or most likely will use whatever schema was used last) ?

Reply:
It uses the public schema, i've tested before :)

Reply:
View:
![image](https://github.com/user-attachments/assets/ee56e78c-bcce-4773-a486-77d7d905fd07)

Template:
![image](https://github.com/user-attachments/assets/0fe28091-4864-466b-8f3a-2f411426385e)

Rendered:
![image](https://github.com/user-attachments/assets/f3b54bd7-d859-4679-855c-4f9490585e2f)

Reply:
Tenants from admin:
![image](https://github.com/user-attachments/assets/3e77851e-85b8-4a15-8fa1-7c2dca15a8e8)